doc/whats_new/v1.6.rst

```diff
@@ -13,6 +13,16 @@ Version 1.6
   For a short description of the main highlights of the release, please refer to
   :ref:`sphx_glr_auto_examples_release_highlights_plot_release_highlights_1_6_0.py`.
 
+..
+  DELETE WHEN 1.6.0 IS RELEASED
+  Starting from v1.6.0, scikit-learn will be using `towncrier` to manage the
+  changelog. Each changelog entry should be it's own file and live in
+  `doc/whats_new/upcoming_changes/`, within the relevant module named folder.
+  The file should be named with the format `<PR_number>.<type>.rst`, and
+  contain a single bullet point changelog entry, with user name but without PR
+  number.
+  See `doc/whats_new/upcoming_changes/README.md` for full details.
+
```

Comment:
I think using `towncrier` is a detail that contributors shouldn't care about. This might be more relevant?

```suggestion
To add your changelog, create a file under the right folder under
`doc/whats_new/upcoming_changes` with file format `<PULL REQUEST>.<TYPE>.rst`, where
`<PULL REQUEST>` is a pull request number, and `<TYPE>` is one of:

* `major-feature`
* `feature`
* `efficiency`
* `enhancement`
* `fix`
* `api`

Example:

- :class:`ensemble.ExtraTreesClassifier` and :class:`ensemble.ExtraTreesRegressor`
  now supports missing values in the data matrix `X`. Missing-values are
  handled by randomly moving all of the samples to the left, or right child
  node as the tree is traversed.
  By :user:`Adam Li <adam2392>`

  See `doc/whats_new/upcoming_changes/README.md` for full details.

```

Reply:
It might be a good idea to mention that some updates (e.g., changes to private methods) don't require a changelog entry.

Reply:
@adrinjalali done. I mostly wanted to include the fact that there was a change in how we do changelogs, for people fixing merge conflicts in that file and thinking that they have done something wrong (like I did :sweat_smile: ). I included a sentence that just says we changed things. Happy to amend.

Reply:
I think you missed the newlines when you copy pasted the code.